tests/console/yast2_clone_system.pm

```diff
@@ -44,22 +44,19 @@ sub run {
 
     unless (is_opensuse) {
         my $devel_repo = get_required_var(is_sle('>=15') ? get_repo_var_name("MODULE_DEVELOPMENT_TOOLS") : 'REPO_SLE_SDK');
```

Comment:
why did you decide to hide factory repo in this variable? shouldn't be tricky now to setup in openQA? just asking, I'm not sure, hardcoding it maybe more clear.

Reply:
@b10n1k  Please, do not resolve conversations that are not addressed somehow.
@rwx788 do you know what is the preference to override settings when they are coming from the repo synchronization and where this setting could be located?

Reply:
Checked with @rwx788 that we should not try to override variable here, as when the build is triggered it will be overriden, so we need new variable, for example `JING_REPO` and we would be able to apply it better where we need it.